pkg/cmd/uninstall/uninstall.go

```diff
@@ -245,3 +253,27 @@ func (o *UninstallOptions) DeleteReleaseIfPresent(ns string, releaseName string,
 	}
 	return errors
 }
+
+func (o *UninstallOptions) setPVCFinalizerToNull(ns string, releaseName string) error {
```

Comment:
_releaseName_ is not really used. I think `setPVCFinalizerToNull` is really a "global" function to be executed somewhere from within `Run`.